DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/GroupCoursesController.cs

```diff
@@ -63,7 +62,7 @@ public IActionResult Index(int groupId, int page = 1)
                 searchSortPaginationOptions
             );
 
-            var model = new GroupCoursesViewModel(groupId, groupName!, result);
+            var model = new GroupCoursesViewModel(groupId, result);

```

Comment:
I've added this to the h1, as we have similar h1s elsewhere that have the names plonked in them. I think some of the field editing for progress details are examples. I think it is fine here, but can look perhaps a bit off with longer group names because of the heading button. Happy to change it to something else if you have a preference. Example:
![image](https://user-images.githubusercontent.com/80777689/161972023-75a116bd-f0ad-4923-a39b-79762707fb4b.png)

Reply:
I was thinking more like this
![image](https://user-images.githubusercontent.com/59561751/161972777-9789ec0b-22d0-4f5e-994f-094a7222f85c.png)

Reply:
The group name applies to both tabs, so I think it belongs above them

Reply:
Fair point. I was thinking it would look a little funny, but it doesn't look too bad.
![image](https://user-images.githubusercontent.com/80777689/161975273-267227b9-c0f2-4ef6-9142-90fd19ae4584.png)

Reply:
That was just because of what I cut and pasted into the screenshot in paint. Yeah that looks a lot better